### PR TITLE
Add DPLL event subscription for  multiple cards

### PR DESCRIPTION
**Pull request by aneeshkp**
This PR extends DPLL  event subscription to  multiple card configured under WPC config .

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -18,13 +19,51 @@ const (
...
+func (i *IFaces) GetEventSource(iface string) event.EventSource {
+	for _, ii := range *i {
+		if ii.Name == iface {
+			return ii.Source
+		}
+	}
+	return event.PTP4l
+}
+
+// GetEventSource ... get event source
```
Comment (jzding) [inline pkg/config/config.go:55]: fix comment

File: pkg/daemon/config.go [hunk 2]
```diff
@@ -176,31 +176,26 @@ func (output *ptp4lConf) populatePtp4lConf(config *string) error {
...
 
-func getSource(is_ts2phc_master string) (source event.EventSource, err error) {
-	if master, err := strconv.ParseBool(strings.TrimSpace(is_ts2phc_master)); err == nil {
-		if master {
-			return event.GNSS, nil
-		} else {
-			return event.PPS, nil
+func getSource(is_ts2phc_master string) event.EventSource {
+	if ts2phcmaster, err := strconv.ParseBool(strings.TrimSpace(is_ts2phc_master)); err == nil {
+		if err != nil || ts2phcmaster {
```
Comment (jzding) [inline pkg/daemon/config.go:181]: `err != nil` won't be true since it is in the path of  `err == nil` from previous line

Comment (aneeshkp) [reply pkg/daemon/config.go]: done

File: pkg/config/config.go (hunk 1 above)
Comment (aneeshkp) [reply pkg/config/config.go]: done

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**